ssl-config/src/main/java/com/palantir/remoting/ssl/SslSocketFactories.java

```diff
@@ -63,7 +63,7 @@ public static SSLContext createSslContext(SslConfiguration config) {
         }
     }
 
-    private static TrustManagerFactory createTrustManagerFactory(
+    public static TrustManagerFactory createTrustManagerFactory(
```

Comment:
the trustStorePath parameter is non-trivial (can be file or directory, depending on the type), so this will need docs, unfortunately.